src/korma_encrypted/core.clj

```diff
@@ -35,15 +35,17 @@
 
 (defn encrypted-field [ent field]
   (-> ent
-      (korma/prepare (fn [v]
-                       (let [unencrypted (get v field)]
-                         (-> v
-                             (assoc (encrypted-name field) (encrypt-value unencrypted))
-                             (dissoc field)))))
-      (korma/transform (fn [v]
+      (korma/prepare (fn [values]
```

Comment:
@wjdix Just pushed a new commit that refactors those out into new methods. Also made the testing a lot nicer. Let us know what you think.